pkg/apis/nmstate/v1alpha1/nmstate_types.go

```diff
@@ -0,0 +1,28 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NMstate is the Schema for the nmstates API
+// +kubebuilder:subresource:status
+// +kubebuilder:resource:path=nmstates,scope=Namespaced
```

Comment:
I think scope=Cluster for this, @phoracek is that right ?

Reply:
It is. This operator can be deployed only once, so let's make it clear that it is a cluster operator

Reply:
Thanks. Good catch.